sinon/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Sinon 1.16.0
+// Type definitions for Sinon 2.1.0
 // Project: http://sinonjs.org/
-// Definitions by: William Sears <https://github.com/mrbigdog2u>
+// Definitions by: William Sears <https://github.com/mrbigdog2u>, Jonathan Little <https://github.com/rationull>
```

Comment:
Do you want to be a reviewer on future changes to SinonJS? The original author only contributed the initial 3 commits (over 4 years ago), so I think it's fine to add more reviewers.